### DISCUSSION TITLE
Typescript sources is not compiled yet 에러 해결 방법이 있을까요?

### DISCUSSION THREAD

**Discussion by alcnlsss** [naverz/zepeto-studio-kor#117] [5. Q&A]
안녕하세요.
Typescript sources is not compiled yet 오류에 대해 질문있습니다.

zepeto측에 수없이 문의하여 많은 답변을 받았으나 저 오류를 해결하지 못하여 진행자체에 어려움을 겪고있습니다...
혹시 해결하신분이 있으신지 있으시다면 방법을 여쭈어보려고 합니다.

샘플을 다운받아서 실행했을때는 이상없이 실행되나 Typescript파일을 열어서 수정하거나 새로 Typescript파일을 만들면 저 오류가 나타납니다. 

그리고 샘플 실행 시 로그 창에 'Server Can't Start'라는 에러 로그가 출력되는데 이부분이 문제가 있는걸까요?..

![오류이미지1](https://user-images.githubusercontent.com/34666534/147038392-19a5d1e0-4256-4d94-9ba0-8307b62278d6.jpg)

![오류이미지2](https://user-images.githubusercontent.com/34666534/147038432-da49bbfa-7307-4f11-83bc-5809003369f3.jpg)
그리고 기존 컴파일된 파일을 수정하면 위 그림처럼 Compiled가 활성화 되지 않습니다..

zepeto측에서 답변해주신 방법들은
1. 메뉴 - ZEPETO - Script - Rebuild All (리빌드하였으나 동일증상)
2. 샘플을 다운받아서 진행 (샘플은 모두 정상작동하나 Typescript파일을 수정하거 새로만들면 동일증상)
3. node.JS제거 후 재설치, 방화벽 설정에서 node.exe와 Unity Editor를 허용 (동일증상)
4. 메뉴 - ZEPETO > Script > Experimental > Refresh Environments (동일증상)
5. 메뉴 - Window -> Pakcage Manager -> zepeto.world (업데이트 후 동일증상)
6. 해당 유니티 프로젝트 패키지 경로에서 Libaray 폴더 안에 node 폴더가 있는지 확인 (폴더 존재 확인, 동일증상)
이렇게 해보았지만 증상이 해결되지 않는 어려움입니다.

이 외에 방법을 아신다면 도움부탁드립니다!!

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 위 내용 재현이 어려운데 최신 월드에서도 같은 현상이 나타나는지 확인 부탁드립니다.
감사합니다.
